ci-operator/config/stackrox/stackrox/stackrox-stackrox-nightlies.yaml

```diff
@@ -31,73 +31,6 @@ resources:
       memory: 4000Mi
 test_binary_build_commands: .openshift-ci/dispatch.sh test-binary-build-commands
 tests:
```

Comment:
(not about this line)

@janisz
What's the story/plan for Nightly tests in GHA?
I wasn't able to find nightly scheduled workflow in our `.github` directory and so removing tests from OSCI means we'll be without some coverage for nightlies. I think, removing these nightly tests is premature and first we should enable them in GHA, which requires setting up a second Slack notifier for GHA to post in `#nightly-ci-runs`.

Reply:
We run tests on nightlies e.g.: https://github.com/stackrox/stackrox/actions/runs/4190913726/jobs/7264814327

Reply:
Can you please point me how is it set up?

Reply:
https://github.com/stackrox/stackrox/actions/runs/4190913726/workflow#L4-L5

Nightly is a tag on an empty commit and we run tests on push on tags. Maybe we should also migrate the job that creates this commit.

Reply:
Related: https://github.com/stackrox/stackrox/pull/4908. This will get the slack messages for merge failures. It should also run for a tag push i.e. the night trigger but messages go to #test-failures.